### PR TITLE
Check for nonexistent time and project when updating.

### PR DESCRIPTION
**Pull request by LyonesGamer**
A quick solution to ameliorate #270 (and something we should have been doing this whole time *anyway*), but it won't ultimately be the fix.

### CODE REVIEW

**Comment by Kennric** [comment]
That looks ok to me, simple short-circuit error +1

**Comment by ElijahCaine** [comment]
@LyonesGamer Is there any way to add a regression test for this?

**Comment by Kennric** [comment]
I think those tests are correct, get a second opinion, though. +1

File: tests/times.js [hunk 1]
```diff
@@ -4127,19 +4127,23 @@ module.exports = function(expect, request, baseUrl) {
      * Without this function you would see this exact code pretty 26
      * times over.
      */
-    function checkPostToEndpoint(done, postObj, expectedResults, error,
+    function checkPostToEndpoint(done, uri, postObj, expectedResults, error,
```
Comment (ElijahCaine) [inline tests/times.js]: Why are we passing uri to this function now? iirc it doesn't get used.

**Comment by ElijahCaine** [comment]
Tests look good. +1 from me. Merge it @LyonesGamer

**Pull request merged**